<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–°–∫–ª–∞–¥—Å–∫–∞—è —Å–∏—Å—Ç–µ–º–∞ HK</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f0f5ff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>–ë—ã—Å—Ç—Ä—ã–π –¥–æ—Å—Ç—É–ø</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="warehouseButton">
         <property name="text">
          <string>üì¶ –°–∫–ª–∞–¥</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #1976d2;
    color: white;
    border: none;
    padding: 12px 24px;
    border-radius: 6px;
    font-size: 12pt;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #1565c0;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="requestsButton">
         <property name="text">
          <string>üìã –ó–∞—è–≤–∫–∏</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #1976d2;
    color: white;
    border: none;
    padding: 12px 24px;
    border-radius: 6px;
    font-size: 12pt;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #1565c0;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reportsButton">
         <property name="text">
          <string>üìä –û—Ç—á–µ—Ç—ã</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #1976d2;
    color: white;
    border: none;
    padding: 12px 24px;
    border-radius: 6px;
    font-size: 12pt;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #1565c0;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="usersButton">
         <property name="text">
          <string>üë• –ü–æ–ª—å–∑–æ–≤–∞—Ç–µ–ª–∏</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #1976d2;
    color: white;
    border: none;
    padding: 12px 24px;
    border-radius: 6px;
    font-size: 12pt;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #1565c0;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settingsButton">
         <property name="text">
          <string>‚öôÔ∏è –ù–∞—Å—Ç—Ä–æ–π–∫–∏</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #1976d2;
    color: white;
    border: none;
    padding: 12px 24px;
    border-radius: 6px;
    font-size: 12pt;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #1565c0;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>üö™ –í—ã—Ö–æ–¥</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #d32f2f;
    color: white;
    border: none;
    padding: 12px 24px;
    border-radius: 6px;
    font-size: 12pt;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #c62828;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>